zeros:
  gimbal1: 0
  gimbal2: 0
  gimbal3: 0
  gimbal4: 0
  gimbal5: 0
  gimbal6: 0
